spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageProtocolResolver.java

```diff
@@ -36,21 +36,26 @@
  * @author Vinicius Carvalho
  * @author Artem Bilan
  */
-public class GoogleStorageProtocolResolver implements ProtocolResolver, BeanFactoryPostProcessor, ResourceLoaderAware {
+public class GoogleStorageProtocolResolver
+		implements ProtocolResolver, BeanFactoryPostProcessor, ResourceLoaderAware {
 
 	private static final Log logger = LogFactory.getLog(GoogleStorageProtocolResolver.class);
 
 	private static final String PROTOCOL = "gs://";
 
 	private ConfigurableListableBeanFactory beanFactory;
 
+	private GoogleStorageProtocolResolverSettings settings;
+	private boolean settingsAvailable = true;
```

Comment:
I guess we can have something like `settings = new GoogleStorageProtocolResolverSettings();` with all the default possible options.

That way we don't need to require from end-user to specify such a bean in his/her application.
Therefore no reason to warn about that bellow.
The `info()` is OK though with the note that we are going to use all the default settings.